README.md

```diff
@@ -142,7 +142,7 @@ Thank you to all the contributors to this one of [many open source projects at S
 ### Active Key Contributors
 
 * [@davidlygagnon](https://github.com/davidlygagnon) David Ly-Gagnon
-* [@futuremint](https://github.com/futuremint) David Woodward
```

Comment:
I guess I'll let @futuremint approve this one! His official name is David though.

Reply:
Oh I see, it was named Dave in all his other handles 😆